### PR TITLE
Object oriented MPL for ep 7

### LINKED ISSUE #111: Use object oriented matplotlib
See https://github.com/hsf-training/hsf-training-matplotlib/issues/48 for background/instructions.

See [this search](https://sourcegraph.com/search?q=context%3Aglobal+repo%3A%5Egithub%5C.com%2Fhsf-training%2Fhsf-training-matplotlib%24++%28plt.plot+or+plt.hist+or+plt.errorbar+or+plt.yticks%29&patternType=standard) for places that need changing.

This issue is for episode 7.

### PR DESCRIPTION
**Pull request by amanmdesai**
Closes #111

### CODE REVIEW

File: _episodes/07-dimuonspectrum.md [hunk 1]
```diff
@@ -343,14 +345,13 @@ You could use the `np.logspace()` function for the binning. It helps in returnin
...
->plt.hist(pm, bins=logbins, histtype="step")
->
->plt.xlabel("mass (GeV/$c^2$)")
->plt.ylabel("Events")
->plt.xscale("log")
->plt.title("Mass of dimuons per event")
->plt.autoscale()
+>fig, ax = plt.subplots()
+>ax.hist(pm, bins=logbins, histtype="step")
+>ax.xlabel("mass (GeV/$c^2$)")
```
Comment (klieret) [inline _episodes/07-dimuonspectrum.md]: Isn't it `set_xlabel`? Probably best to quickly run this snippet locally to check that it still works :) 
(also not sure about `autoscale`)

**Review by klieret — CHANGES_REQUESTED** [review]
Thanks, this is almost done :)

File: _episodes/07-dimuonspectrum.md (hunk 1 above)
Comment (amanmdesai) [reply _episodes/07-dimuonspectrum.md]: I tested autoscale it worked. 
yeah its set_xlabel (my mistake)

**Comment by amanmdesai** [comment]
Thanks for the review @klieret

File: _episodes/07-dimuonspectrum.md (hunk 1 above)
Comment (amanmdesai) [reply _episodes/07-dimuonspectrum.md]: DONE

**Comment by amanmdesai** [comment]
i think this PR is ready review

**Review by klieret — APPROVED** [review]
LGTM!

**Pull request merged**